src/Silex/Application.php

```diff
@@ -251,7 +251,17 @@ public function before($callback, $priority = 0)
                 return;
             }
 
-            $ret = call_user_func($app['callback_resolver']->resolveCallback($callback), $event->getRequest(), $app);
+            $request = $event->getRequest();
+            $c = $app['callback_resolver']->resolveCallback($callback);
```

Comment:
Could please use a more clear name for the `$c` variable? For example `$resolvedCallback`? Thanks@